### DISCUSSION TITLE
Custom overlay wishlist

### DISCUSSION THREAD

**Discussion by rusty-snake** [Helium314/SCEE#575] [Ideas]
- Specify a commit message in each custom overlay rather than merging all in a very generic message.
- Highlight missing data filter. Use cases are
  - Not highlight data as missing on `access=private|no` but on `access=yes`. The difference to element selection is that existing data is still shown and you can select it to edit it.
  - => #586
- Additional filter for tags shown in bottom-pop-up but not used for highlighting. Use cases are
  - easily seeing tags like `segregated`, `cycleway:surface` without opening tags-editor and without polluting colors
  - easily seeing tags like `surface` without breaking `!tracktype` (i.e. single tag to edit) feature.
  - => #594
- Manual ordering (insert of by creation order)
  - => #587

**Comment by Helium314** [comment]
It would be more useful to open issues if you would like this actually implemented. (and search for duplicates, e.g. commit message exists already #494)

And maybe you want to add a more detailed description on your last point, I don't understand what you want.

**Reply by rusty-snake** [reply]
> commit message exists already https://github.com/Helium314/SCEE/issues/494

As I commented in #494, there are two steps on commit messages
- `Edit user-defined element selection` -> ` Edit user-defined element selection: maxspeed` (message per layer)
- `Edit user-defined element selection` -> `Tempo 30 hinzugefügt` (message per edit/quest)

>  open issues

reasonable, will do

This can still be used for brainstorming and discussions before going to issues.
